ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -733,18 +731,7 @@ export default class ConfirmTransactionBase extends Component {
         this._removeBeforeUnload();
 
         sendTransaction(txData)
-          .then(() => {
-            clearConfirmTransaction();
-            this.setState(
-              {
-                submitting: false,
-              },
-              () => {
-                history.push(mostRecentOverviewPage);
-                updateCustomNonce('');
```

Comment:
It seems like we are resetting the `custom nonce` over here I can see that it is done for the error condition as well, I wonder this is handled with the current changes as well.